docs/diagrams/DeleteExecuteSequenceDiagram.puml

```diff
@@ -0,0 +1,47 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant "e:XYZDeleteCommand" as XYZDeleteCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant ":UniqueGuestList" as UniqueGuestList MODEL_COLOR
+end box
+
+note right of XYZDeleteCommand: XYZ = Guest or Vendor
+[-> LogicManager
+activate LogicManager
+
+LogicManager -> XYZDeleteCommand : execute()
+activate XYZDeleteCommand
+
+XYZDeleteCommand -> Model : deleteXYZ(1)
+activate Model
+note right of XYZDeleteCommand: deleteXYZ = deleteGuest or deleteVendor
+
+Model -> UniqueGuestList : deleteXYZ(1)
+activate UniqueGuestList
+
+UniqueGuestList --> Model
+deactivate UniqueGuestList
+
+Model --> XYZDeleteCommand
+deactivate Model
+
+create CommandResult
+XYZDeleteCommand -> CommandResult
+activate CommandResult
+
+CommandResult --> XYZDeleteCommand
+deactivate CommandResult
+
+XYZDeleteCommand --> LogicManager : result
```

Comment:
Should `XYZDeleteCommand` be deactivated for consistency?

Reply:
oh cos the execution of the XYZCommand only happens in the following diagram, so I left it there for continuity